manifests/primary/firewall.pp

```diff
@@ -5,15 +5,26 @@
 #
 class profile_kerberos_server::primary::firewall {
 
-  # This should be kadmin and kpasswd
+  # kpasswd
   ['tcp','udp'].each |$protocol| {
-
-    firewall { "212 Kerberos via ${protocol}":
+    firewall { "212 kpasswd ${protocol}":
       proto  => $protocol,
-      dport  => [
-        '464',
-        '749',
-      ],
+      dport  => '464',
+      action => 'accept',
+    }
+  }
+
+  # kadmin
+  [
+    '141.142.0.0/16',
+    '10.142.0.0/16',
+    '172.24.0.0/13',
+    '172.16.0.0/13'
```

Comment:
What is this `172.16.0.0/13` subnet? I'm not familiar with that being a NCSA subnet?